ext/net/ops.rs

```diff
@@ -704,7 +705,16 @@ where
     .filter_map(rdata_to_return_record(record_type))
     .collect();
 
-  Ok(results)
+  match results[0] {
+    DnsReturnRecord::Unsupported => {
+      let var: &'static str = record_type.into();
+      Err(custom_error(
+        "NotSupported",
+        var.to_owned() + " records are not supported",
```

Comment:
Are either of these showing up in the test added by this PR?

Reply:
This won't be appear in ts file but will display on Deno CLI, as I checked.